boltdesignsystem/bolt
# Academy-related Background Updates (Support Layers, New Fill Colors, etc)

## Jira
N/A

## Summary
Background component updates needed for Academy + new design treatments.

## Details
- Adds the needed ability for us to add / customize the different layers that render inside the Background component. 
- Splits up Background into smaller pieces that can be called individually (ex. swap the order of layers that render)
- Wires up certain Background props to point to the schema
- Adds a handful of new gradient / `fillColor` combo options introduced in the new Academy designs

## How to Test
- Review updated Academy prototypes (specifically Academy's new landing pages)
- Double check that the Background & Background Shape demos are continuing to work and render as expected
- High level (ie. quick) gut check on the code itself. I'm trying to not make a lot of changes here -- splitting up the Background into smaller pieces that can render separately seemed like a much more sane approach to take vs cramming even more complex logic into an already overly complex Twig template.
